assets/html/warner.js

```diff
@@ -37,7 +37,7 @@ function maybeAddWarning () {
         document.body.removeChild(div);
     });
     const href = window.documenterBaseURL + '/../' + window.DOCUMENTER_STABLE;
-    div.innerHTML = 'This documentation is not for the latest version. <br> <a href="' + href + '">Go to the latest documentation</a>.';
+    div.innerHTML = 'This documentation is for the development version.<br> <a href="' + href + '">Go to the documentation for the latest stable release</a>.';
```

Comment:
Hmm. This is not entirely correct, since the same message also gets displayed on old versions. Maybe just say "this is not for the current latest stable release" or something like that?

Reply:
@ViralBShah What do you think of this?

```suggestion
    div.innerHTML = 'This documentation is not for the latest stable release, but for either the development version or an older outdated release..<br> <a href="' + href + '">Click here to go to the documentation for the latest stable release</a>.';
```

Reply:
I just deleted the word `outdated`, since older release is sufficient.